### PR TITLE
fixes #520 misleading draft actions

### LINKED ISSUE #520: Draft actions are misleading on event creation page
When using the indicative form of verbs on buttons, user might expect something to happen. I'd prefer a more passive version. Moreover, when finally clicking on the create button on the bottom, it is not obvious what will happen since it says "Erstellen" independently of my choice. @hpi-swt2/product-owner-2016 

bildschirmfoto 2017-02-04 um 22 45 32

### PR DESCRIPTION
**Pull request by corinnaj**
We now have actual buttons, instead of radio buttons for drafts. It is not possible to unpublish events, I wasn't sure if that is a useful feature, but it should be possible to reintroduce that quickly, if necessary.

I wasn't sure what our conventions for buttons in tests are, so they might have to be refactored.

### CODE REVIEW

File: spec/features/event_image_spec.rb [hunk 1]
```diff
@@ -19,20 +18,20 @@
...
 
     it "should not change the image if validation for a new image fails" do
-      click_button I18n.t(".events.form.create")
+      click_button "create"
       previous_image = Event.last.image
 
       visit edit_event_path(Event.last)
       attach_file "event[custom_image]", File.join(Rails.root + 'spec/testfiles/too_small_image.png')
-      click_button I18n.t('.events.form.update')
+      click_button "update"
```
Comment (bjrne) [inline spec/features/event_image_spec.rb]: Why did you remove the I18n here?

**Review by bjrne — APPROVED** [review]
I don't know the motivation for removing all the internationalization lines, but if it's legitimate, then this is ready to go 🏖

File: spec/features/event_image_spec.rb (hunk 1 above)
Comment (corinnaj) [reply spec/features/event_image_spec.rb]: That's what I meant about button conventions in tests. You can use either the button label to click the button, which is the I18n line, or the button id, which is "update".
We mostly use the labels currently, but I used the ids in some places, because the labels got quite long.

Comment (bjrne) [reply spec/features/event_image_spec.rb]: Okay. Since this never lets a line exceed 80 chars or even create a new line, I'm in favor of the I18n version, but I'd like more opinions on this here @hpi-schuelerklub/swt2

**Review by bjrne — CHANGES_REQUESTED** [review]
Attention, there are git tags in the latest commit

**Comment by corinnaj** [comment]
I am not sure what you mean... I removed the one remaining git conflict, I think it should be fine now.

File: spec/features/event_image_spec.rb [hunk 2]
```diff
@@ -9,14 +9,9 @@
     before :each do
       login_as(FactoryGirl.create(:user, role: :organizer), :scope => :user)
       visit new_event_path
-<<<<<<< HEAD
```
Comment (bjrne) [inline spec/features/event_image_spec.rb]: Ah, I thought this was green! sorry :D

**Review by bjrne — APPROVED** [review]

**Comment by annkatrinkuessner** [comment]
We decided that our style guideline is to use internationalization for the tests instead of ids. It would be amazing if this could be adapted before merging :)

**Comment by corinnaj** [comment]
I updated all test to use I18n instead of ids. There is only one rubocop issue remaining, which I propose to fix in another PR, as it's more of a general problem and not part of this issue.

**Comment by querenker** [comment]
That the classes are too long is indeed a general problem... The rubocop todo file is now adjusted accordingly :)
